src/test/basics/base_uint_test.cpp

```diff
@@ -68,15 +93,15 @@ struct base_uint_test : beast::unit_test::suite
         {
             BEAST_EXPECT(d == --t);
         }
```

Comment:
👍 the rules for writing test code are slightly different than the rules for writing production code.